public/colors.css

```diff
@@ -27,6 +27,7 @@
   --color-lightgray-tint-10: #e9edf1;
   --color-lightgray: #ced8e1;
   --color-lightgray-shade-10: #b3c3d1;
+  --color-light-shadow-gray: rgba(51, 51, 51, 0.08);
```

Comment:
@MeBrei @rudolfs does this make sense here or in `global`?

Reply:
`color.css` will be overwritten once you run `yarn generate:colors` the next time. I would put it in `general.css` for now, but maybe write an issue in the Housekeeping milestone to include transparent colors in the color generation.